features/external_users/advocate_new_claim.feature

```diff
@@ -1,275 +0,0 @@
-      And I submit to LAA
-     Then There should not be any Initial Fees saved
-      And There should be a Miscellaneous Fee Saved
-
-  @javascript @webmock_allow_localhost_connect
-  Scenario Outline: Daily attendance fees derived from actual length of Trial
-  Given I am a signed in advocate
-    And There are case types in place
-    And I am on the new claim page with Daily Attendance Fees in place
-    And I select2 a Case Type of "Trial"
-    And I fill in actual trial length with <actual_trial_length>
-   Then The daily attendance fields should have quantities <daf_quantity>, <dah_quantity>, <daj_quantity>
-
-  Examples: Actual Trial lengths and how the Daily attendance fields should break it down
-    | actual_trial_length   | daf_quantity | dah_quantity | daj_quantity |
-    | 1                     | 0            | 0            | 0            |
-    | 2                     | 0            | 0            | 0            |
-    | 3                     | 1            | 0            | 0            |
-    | 40                    | 38           | 0            | 0            |
-    | 41                    | 38           | 1            | 0            |
-    | 50                    | 38           | 10           | 0            |
-    | 51                    | 38           | 10           | 1            |
-    | 60                    | 38           | 10           | 10           |
-    | 70                    | 38           | 10           | 20           |
-
-  #TODO could potentially merge this Scenario:with the one above
-  @javascript @webmock_allow_localhost_connect
-  Scenario Outline: Daily attendance fees derived from actual length of Retrial
-  Given I am a signed in advocate
-    And There are case types in place
-    And I am on the new claim page with Daily Attendance Fees in place
-    And I select2 a Case Type of "Retrial"
-    And I fill in actual retrial length with <actual_retrial_length>
-   Then The daily attendance fields should have quantities <daf_quantity>, <dah_quantity>, <daj_quantity>
-
-  Examples: Actual Trial lengths and how the Daily attendance fields should break it down
-    | actual_retrial_length | daf_quantity | dah_quantity | daj_quantity |
-    | 1                     | 0            | 0            | 0            |
-    | 2                     | 0            | 0            | 0            |
-    | 3                     | 1            | 0            | 0            |
-    | 40                    | 38           | 0            | 0            |
-    | 41                    | 38           | 1            | 0            |
-    | 50                    | 38           | 10           | 0            |
-    | 51                    | 38           | 10           | 1            |
-    | 60                    | 38           | 10           | 10           |
-    | 70                    | 38           | 10           | 20           |
-
-  Scenario: Uncalculated fees (PPE and NPW) accepts total amount only
```

Comment:
Let's make a note in a TODO file that this needs to be tested in some sort of javascript testing
 Add a line note